context_test.go

```diff
@@ -341,49 +334,49 @@ func TestDetectProjectGOPATH(t *testing.T) {
 		{
 			name:         "project-with-no-AbsRoot",
 			root:         "",
-			resolvedRoot: filepath.Join(ctx.GOPATHs[0], "src", "real", "path"),
+			resolvedRoot: filepath.Join(ctx.GOPATHs[0], "src/real/path"),
```

Comment:
I'm under the impression that `filepath.Join` will call `filepath.Clean` which calls filepath.FromSlash` on the passed path. Am I missing something else?

Anyhow, this is a purposeless change and I can revert it. :grin: